app/components/logic/pretty-print-time.tsx

```diff
@@ -13,7 +13,13 @@ export const PrettyPrintTime: React.FunctionComponent<PrettyPrintTimeProps> = ({
     time ? new Date(time).toISOString() : "None"
   );
   useEffect(
-    () => setTimeString(time ? new Date(time).toLocaleString() : "None"),
+    () =>
+      setTimeString(
+        (time ? new Date(time).toLocaleString() : "None") +
+          " (local time), " +
```

Comment:
```suggestion
          ", " +
```

I think we can get away without saying local time now that we have the "ago" thing -- just thinking out loud, curious your thoughts

Reply:
makes sense to me